test/test_shapes.py

```diff
@@ -93,8 +93,24 @@ def test_n(self):
     def test_plt(self):
         t = tadasets.swiss_roll(n=345)
         tadasets.plot3d(t)
-            
+
     def test_ambient(self):
         s = tadasets.swiss_roll(n=200, ambient=15)
         assert s.shape == (200, 15)
 
+class TestInfty:
+    def test_n(self):
+        t = tadasets.infty_sign(n=345)
+        assert t.shape[0] == 345
+
+    def test_rotation(self):
+        try:
```

Comment:
For the testing here, checkout pytest's `pytest.raises` method documented here https://docs.pytest.org/en/stable/assert.html

Reply:
Done!